main.go

```diff
@@ -218,15 +369,18 @@ func (c *controller) worker() {
 }
 
 func (c *controller) sync() {
+	c.reconcileAttempts.Inc()
 	configMap, ok, err := c.cmapInf.GetStore().GetByKey(fmt.Sprintf("%s/%s", c.options.namespace, c.options.configMapName))
 	if !ok || err != nil {
+		c.reconcileErrors.With(prometheus.Labels{"type": "fetch"}).Inc()
```

Comment:
prefer `.WithLabelValues` in these cases (https://github.com/prometheus/client_golang/blob/ff1d4e21c12e4c0b190a166d96507786c8f989d6/prometheus/counter.go#L203), also I think it would be good to move these values to constants